tests/default/indices/mapping/index.yaml

```diff
@@ -145,4 +145,23 @@ chapters:
     response:
       status: 200
       payload:
-        acknowledged: true
\ No newline at end of file
+        acknowledged: true
+  - synopsis: Add or update mapping using POST method.
```

Comment:
This is a bit redundant with what's above. Change the first `PUT /{index}/_mapping` to having two verbs, `PUT` and `POST`.